frontend/src/components/Project/index.tsx

```diff
@@ -0,0 +1,56 @@
+import './project.css';

```

Comment:
again, why import and create this file if it isnt being used?

Reply:
The idea was I go ahead and setup the stuff for a css file in case it's implemented later, I saw it being done elsewhere in our project so I just went with it